### PR TITLE
Add binding for metric

### LINKED ISSUE #28: Bindings for Metrics

### PR DESCRIPTION
**Pull request by jihchi**
Fix #28

### CODE REVIEW

**Comment by zploskey** [comment]
Some of these values should probably be typed as `int`, don't you think?

**Comment by jihchi** [comment]
I'm not sure which of metrics should be `int` type

https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pagemetrics

```
Timestamp <number> The timestamp when the metrics sample was taken.
Documents <number> Number of documents in the page.
Frames <number> Number of frames in the page.
JSEventListeners <number> Number of events in the page.
Nodes <number> Number of DOM nodes in the page.
LayoutCount <number> Total number of full or partial page layout.
RecalcStyleCount <number> Total number of page style recalculations.
LayoutDuration <number> Combined durations of all page layouts.
RecalcStyleDuration <number> Combined duration of all page style recalculations.
ScriptDuration <number> Combined duration of JavaScript execution.
TaskDuration <number> Combined duration of all tasks performed by the browser.
JSHeapUsedSize <number> Used JavaScript heap size.
JSHeapTotalSize <number> Total JavaScript heap size.
```

How about change all metric's type to `int`?

**Comment by zploskey** [comment]
JavaScript does not distinguish between integers and floats so they will show everything as Number. Some of these are naturally integers because they are counts. I think the follow should be integers:

```
Documents <number> Number of documents in the page.
Frames <number> Number of frames in the page.
JSEventListeners <number> Number of events in the page.
Nodes <number> Number of DOM nodes in the page.
LayoutCount <number> Total number of full or partial page layout.
RecalcStyleCount <number> Total number of page style recalculations.
JSHeapUsedSize <number> Used JavaScript heap size.
JSHeapTotalSize <number> Total JavaScript heap size.
```

As a general rule if there is any chance something might overflow a 32 bit int we should type it as float. I don't think the above fields will but please point out if you don't think that is true for any of them.

**Comment by jihchi** [comment]
Thank you and fixed.

File: src/Metrics.re [hunk 1]
```diff
@@ -1 +1,16 @@
...
+  .
+  "Timestamp": float,
+  "Documents": int,
+  "Frames": int,
+  "JSEventListeners": int,
+  "Nodes": int,
+  "LayoutCount": int,
+  "RecalcStyleCount": int,
+  "LayoutDuration": int,
+  "RecalcStyleDuration": int,
```
Comment (zploskey) [inline src/Metrics.re]: LayoutDuration and RecalcStyleDuration should be floats still. I assume they need to be sub-second timings.

Comment (jihchi) [reply src/Metrics.re]: Fixed in commit e35add0

**Pull request merged**